### ISSUE TITLE
ShipIt: How to support git LFS and bitbucket

### ISSUE THREAD

**Issue by itsdouges** [adeira/universe#6189]
Hey mate! What's your thoughts on supporting git LFS, how would we do it? And for Bitbucket to be the "origin" and Github as the "destination" - is there anything special we need to do outside of updating it to understand bitbucket URLs?

**Comment by mrtnzlml** [comment]
> What's your thoughts on supporting git LFS, how would we do it?

I am fine with it but I personally do not use it. A good start might be to check the implementation of [`facebook/fbshipit`](https://github.com/facebook/fbshipit/blob/f99b6ade3dcaa64537e28002a358cbded94d844c/src/shipit/repo/ShipItRepoGIT.php#L473).

> for Bitbucket to be the "origin" and Github as the "destination"

Does origin in this context mean monorepo? In that case, I think you don't need to do anything other than configure your CI in Bitbucket to start exporting to GitHub. Importing from GitHub is already natively supported.

If you mean it vice versa (that is, GitHub is the monorepo, and Bitbucket has the child repositories), then I think we need to update Importit a little bit. Otherwise, it should work fine. That is because Shipit works with Git and doesn't care that much about platforms such as GitHub, GitLab, or Bitbucket.

**Comment by itsdouges** [comment]
Yes exactly bitbucket (or stash) would be the source and GitHub the destination. That's encouraging! I'll have to look at the ShipIt code for git lfs.

**Comment by mrtnzlml** [comment]
@itsdouges Please, let me know in case you discover any incompatibility. I would be curious to know. 😎
